_submissions/round-12/1/concept-map-simple-git.md

```diff
@@ -0,0 +1,14 @@
+date: 2015-02-25
+round: Round 12
+title: Simple version control with git commands
+author: Stephanie Mark
+permalink: /2015/02/stephanie-mark/
+tags:
+  - Concept Map
+  - git
+
+Link to [Version Control with Git](http://swcarpentry.github.io/git-novice/01-backup.html) lesson on Software Carpentry.
+
+The part of the lesson I've mapped is simple version control with git via the command line.
+
+[Concept Map](http://imgur.com/XEWX2XT)
```

Comment:
The link will need to be `http://i.imgur.com/XEWX2XT.jpg` to pick up the actual image (rather than the page containing the image).
Thanks,
@gvwilson